### ISSUE TITLE
How to list all projects and delete from cli

### ISSUE THREAD

**Issue by fiftify** [aws/awsmobile-cli#187]

**Comment by UnleashedMind** [comment]
The awsmobile-cli does not have the command to list all projects. 
It does however have a `delete` command that if executed in a valid awsmobile-cli initialized project, will delete the backend Mobile hub project.

**Comment by fiftify** [comment]
@UnleashedMind does it delete all associated awsmobile features i.e the lambda functions etc.

**Comment by UnleashedMind** [comment]
No, some aws resources are not deleted, such as S3 buckets, you will need to manually clean up those.
